src/zpiv/zpivwallet.cpp

```diff
@@ -12,7 +12,7 @@
 #include "zpivchain.h"
 
 
-CzPIVWallet::CzPIVWallet(std::string strWalletFile)
```

Comment:
zpivwallet has access to chainparams.
why passing this boolean argument to the constructor, instead of getting it as local variable?

Reply:
yeah, was over-thinking it, have re-worked it now